app/views/layouts/application.html.erb

```diff
@@ -1,17 +1,94 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>RouteinesInit</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WhatsNext</title>
```

Comment:
The title is Routines NOT WhatsNext

Reply:
Done

Reply:
Ehm... something doesn't add up here. Please come back to this!